### DISCUSSION TITLE
Add support for signed asset urls

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#4452] [Feature Requests]
Would be nice if we add support for signed asset urls (just like s3) so that the permission to access an asset is scoped to that asset only (and perhaps time constrained)

This allows assets to be shared directly from Directus without having to make all assets public and also avoids people from mistakenly exposing their access tokens when copying a link from Directus app.

**Comment by workatease** [comment]
I agree with @WoLfulus  this looks like a promising  feature will be used to reduce the bandwidth and secure the app better

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
